### PR TITLE
Fix/removedpolicies

### PR DESCRIPTION
**Pull request by dgwhited**
Adds the capability to also post removed policies.

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -47,14 +47,27 @@ def compare_policies():
...
+        # List comprehension to determine if any policies were removed
+        removed_policies = [ policy for policy in old_policies if policy not in current_policies ] 
+
+        # Create lists of strings to post - constraints/ at the beginning of each string is redundant, so it is removed
+        new_policies_to_post = []
+        removed_policies_to_post = []
+        if new_policies:
+            new_policies_to_post = [ f"New Organization Policy Detected: {policy.split('constraints/')[-1]}" for policy in new_policies ]
+        if removed_policies:
+            removed_policies_to_post = [ f"Organization Policy Removed Detected: {policy.split('constraints/')[-1]}" for policy in removed_policies ]
```
Comment (jdyke) [inline src/main.py:61]: Noodled over this a little, IMO:

`Organization Policy Removed Detected` -> `Organization Policy Removal Detected`

**Comment by jdyke** [comment]
This looks great. I like the consistency in arguments you added. Since we are now accounting for policy removals, what are your thoughts on changing the github branch name?

https://github.com/ScaleSec/gcp_org_policy_notifier/blob/00ca33c1b026d1b6683b417f6d75a78a3052eefb/src/main.py#L275

File: src/main.py (hunk 1 above)
Comment (dgwhited) [reply src/main.py:61]: Yeah, I like that - the other thought I had was to put the effect first for consistency

New Organization Policy Detected
Removal of Organization Policy Detected

Comment (jdyke) [reply src/main.py:61]: Love that, lets roll with it!

**Comment by dgwhited** [comment]
> This looks great. I like the consistency in arguments you added. Since we are now accounting for policy removals, what are your thoughts on changing the github branch name?
> 
> https://github.com/ScaleSec/gcp_org_policy_notifier/blob/00ca33c1b026d1b6683b417f6d75a78a3052eefb/src/main.py#L275
This change is in 85add00

File: src/main.py (hunk 1 above)
Comment (dgwhited) [reply src/main.py]: f2bfe37

**Pull request merged**